bin/glite-info-update-endpoints

```diff
@@ -89,13 +95,32 @@ def usage():
               -v --verbose Run in verbose mode
 
             """
+
+def get_url_data(url):
```

Comment:
Is there test coverage for this function?

Reply:
Fair enough! This needs to be added (test coverage), but not by this PR, rather by the maintainers. Thanks for highlighting the need for this